.github/workflows/test.yml

```diff
@@ -0,0 +1,36 @@
+name: Test
+
+on:
```

Comment:
I've started doing this config instead:

```yaml
on:
  pull_request:
  push:
    branches:
      - main
```
So we get it on all manual pushes to the primary branch, and then all PRs to any branch.